docs/index.md

```diff
@@ -171,6 +175,64 @@ Once successfully linked and rebuilt, your application will be connected to Fire
 If you're using an older version of React Native without autolinking support, or wish to integrate into an existing project,
 you can follow the manual installation steps for [iOS](/install-ios) and [Android](/install-android).
 
+---
+## Expo
+
+Integration with Expo is possible in both bare workflow and [custom managed workflow](https://docs.expo.io/workflow/customizing/) via [config plugins](https://docs.expo.io/guides/config-plugins/).
+
+React Native Firebase cannot be used in the "Expo Go" app, because [it requires custom native code](https://docs.expo.io/workflow/customizing/).
+
+### Expo Bare Workflow
+
+If you're using [Bare Workflow](https://docs.expo.io/introduction/managed-vs-bare/#bare-workflow), please follow the above [Android Setup](/#2-android-setup) and [iOS Setup](/#3-ios-setup) steps.
+
+### Expo Managed Workflow
+
+You must use a [custom development client](https://docs.expo.dev/clients/getting-started/). If starting a new app, you can run `npx create-react-native-app -t with-dev-client` to have this set up automatically. It will also allow you to use the Expo Application Service (EAS Build) do the Android and iOS builds.
```

Comment:
Technically, using a dev client is not required. One could also choose to rebuild and reinstall their app after each change. Obviously it's strongly recommended to use a dev client, but I would slightly adjust the wording here to now use "must".

Reply:
Maybe this is a terminology thing, but by "custom dev client" I am differentiating from "Expo Go".  You MUST use a custom dev client.  Expo Go will not work with RNF.

I don't understand the difference between "custom dev client" (which the Expo docs call it and I linked to) vs. "their app" ?